nuggetsapp/utils.py

```diff
@@ -0,0 +1,6 @@
+import datetime
+
+def date_for_x_days_before_today(dayCount): 
```

Comment:
Haha, yeah i feel ya, this is a hard util method to name.  For things like this, it's worth commenting a bit about what this does with a simple example.